### PR TITLE
Softlayer - Run script in process create node and floating_ip Open Stack

### PR DESCRIPTION
**Pull request by ghost**
Added post_uri parameter to the SoftLayer driver. Run script in process create node

### CODE REVIEW

File: libcloud/compute/drivers/softlayer.py [hunk 1]
```diff
@@ -306,6 +308,10 @@ def create_node(self, **kwargs):
         if kwargs.get('ex_local_disk') is False:
             local_disk = 'false'
 
+        post_uri = ''
```
Comment (tonybaloney) [inline libcloud/compute/drivers/softlayer.py:311]: can post_uri be submitted with an empty value? Are there any tests to support this?

**Comment by tonybaloney** [comment]
thanks for the contribution, post_install_script_uri might be a more obvious kwarg name, but thats no biggie. I've added a note with regards to testing absent values.

File: libcloud/compute/drivers/softlayer.py [hunk 2]
```diff
@@ -278,6 +278,8 @@ def create_node(self, **kwargs):
         :type       ex_os: ``str``
         :keyword    ex_keyname: The name of the key pair
         :type       ex_keyname: ``str``
+        :keyword    post_uri: Post-install script to run after reload
```
Comment (Kami) [inline libcloud/compute/drivers/softlayer.py:281]: It's an extension (provider specific / non-standard) argument, so please prefix it with `ex_`.

Comment (Kami) [inline libcloud/compute/drivers/softlayer.py:281]: Some more documentation for this argument (and ideally example in the docs) would also be nice.

Right now it's not really clear what the value should contain - it seems like it should be a full URL to a file or something else?

**Comment by Kami** [comment]
+1 on @tonybaloney's comment and thanks for the contribution :)

**Comment by tonybaloney** [comment]
@onecloudportal @marcydoty @jvitorsf please see previous comment, if you can please make the changes I'd be happy to merge.

**Comment by sercanyaldiz** [comment]
Hi when is this request merged?

**Comment by sercanyaldiz** [comment]
@tonybaloney, is it possible to assign the public/private vlan during the vs creation?

**Pull request closed without merging**